aiohttp/web.py

```diff
@@ -479,7 +479,7 @@ def run_app(
     reuse_port: Optional[bool] = None,
 ) -> None:
     """Run an app locally"""
-    loop = asyncio.get_event_loop()
+    loop = asyncio.new_event_loop()
```

Comment:
Should we also accept a `loop` arg to preserve the ability of the users to use the old loop?
I have a feeling that this change affects the tests (because the neighboring PR https://github.com/aio-libs/aiohttp/pull/5576 submitted after this one passes and restarting the jobs in this one consistently fails).

Reply:
For example, this test https://github.com/aio-libs/aiohttp/blob/8c82ba11b9e38851d75476d261a1442402cc7592/tests/test_run_app.py#L90-L103 relies on the patched loop being pre-set (as well as many other tests rely on the `loop` fixture IIRC but will probably not be affected because they don't use `run_app()`).